docs/docs/themes/using-multiple-gatsby-themes.md

```diff
@@ -32,14 +32,14 @@ module.exports = {
 }
 ```
 
-In the default setup, a blog will be served from the root path (`/`), and the notes content will be served from `/notes`.
+En la configuración predeterminada, se servirá un blog desde la ruta raíz (`/`), y el contenido de las notas se servirá desde `/notes`.
 
-Run `gatsby develop` to start a development server and view your the site:
+Ejecute `gatsby develop` para iniciar un servidor de desarrollo y ver si es el sitio:
 
 ![The homepage of the site created by gatsby-theme-starter](../images/gatsby-theme-starter-home.png)
 
 ![The `notes` route of a site created by gatsby-theme starter](../images/gatsby-theme-starter-notes.png)
 
-## Tutorial
+## El tutorial
 
-For a step-by-step tutorial, see the ["Using Multiple Themes Together" tutorial](/tutorial/using-multiple-themes-together).
+Para ver un tutorial paso a paso, consulte el tutorial ["Uso de varios temas juntos"](/tutorial/using-multiple-themes-together).
```

Comment:
```suggestion
Para ver un tutorial paso a paso, consulta el tutorial ["Usar varios temas juntos"](/tutorial/using-multiple-themes-together).
```

Reply:
Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.